docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -1745,14 +1745,14 @@ config at https://github.com/rackerlabs/autoscaling-chef/blob/master/site-cookbo
                 </note>
             </para>
         </section>
-        <section xml:id="audit-log" security="writerOnly">
+        <?section xml:id="audit-log" security="writerOnly">
             <title>Audit logs</title>
             <para>Because of the deferred nature of Auto Scale, most
                 errors occur outside of the user's session. Errors are
                 logged to a user's cloud feed as an audit log. The
                 audit log contains all the events that have occurred
                 within a user's scaling group. </para>
-        </section>
+        </section>?>
```

Comment:
I'm not sure what this does, but I'm pretty sure it's invalid XML so I doubt it's being parsed the way you expect it. I may be wrong, though!